docs/blockchain_schema/version1.ts

```diff
@@ -0,0 +1,61 @@
+// Version 1 schema
+
+type Multihash = string;
+type HashPointer = string;
+type UID = string;
```

Comment:
I was using the convention in rahafoundation/raha-web-app to just capitalize the first letter, like `Uid`, so for consistency we can just stick with that